packages/p2p/@deriv-stores.d.ts

```diff
@@ -0,0 +1,10 @@
+import type { TStores } from '@deriv/stores';
+import type RootStore from './src/stores/index';
+
+declare module '@deriv/stores' {
+    export function useStore(): TStores & {
+        modules: {
```

Comment:
@ameerul-deriv I don't think we have `p2p` stores attached to `modules` in `core` 🤔 Do we? 🤔 

If not then we can remove this file 🙇🏻

Reply:
i think carol added this for types necessity so we can recognise the deriv stores in p2p, but idk if we need this necessarily or not

Reply:
@ameerul-deriv If you are sure no other package uses `p2p` stores outside of `p2p` itself then we can remove it.

Reply:
alright, ill check once more to see, but i believe no other package uses p2p